test/recipe-test.js

```diff
@@ -118,3 +120,23 @@ describe('recipe', () => {
     assert.deepEqual(noInstructions, [])
   })
 });
+
+describe('filterRecipes', () => {
+  it('should filter list of recipes based on single tag', () => {
+    let stringifiedExpectedRecipes = JSON.stringify([sampleRecipeData[0]]);
+    let filteredRecipes = filterRecipes(sampleRecipeData, 'antipasto')
+    expect(JSON.stringify(filteredRecipes)).to.equal(stringifiedExpectedRecipes);
+  });
+
```

Comment:
Is there a reason we are stringifying items rather than testing with deep equal? 
I checked and the tests all pass with deep equal, which is easier to read since we get that built in functionality with chai

Reply:
I tried to do deep equal first wtf!!! I tried `to.deep.equal` and `to.equal` which are supposed to be equivalent but weren't making it as deep into the object as it needed to go to confirm equality.

What deep equal did you use @lauraguerra1 ?